README.md

```diff
@@ -1,212 +1,217 @@
-        vars:
-          ansible_become: true
-  ```
-3. Execute the following command if the collection is installed from Ansible Automation Hub:
-  ```shell
-  ansible-playbook -i inventory play.yml
-  ```
-4. Execute the following command if you're running from the collection repository checked out locally:
-  ```shell
-  ANSIBLE_ROLES_PATH="roles/" ansible-playbook -i inventory play.yml
-  ```
-
-### Add the root CA that was created to your local truststore.
-
-The certificate can be downloaded from the browser Certificate Viewer by navigating to `https://rekor.<base_domain>`.
-Download the _root_ certificate that issued the Rekor certificate.
-In Red Hat based systems, the following commands will add a CA to the system truststore.
-
-```shell
-$ sudo openssl x509 -in ~/Downloads/root-cert-from-browser -out tas-ca.pem --outform PEM
-$ sudo mv tas-ca.pem /etc/pki/ca-trust/source/anchors/
-$ sudo update-ca-trust
-```
-
-## Signing a Container
+* https://rekor.`<base_hostname>`
+* https://fulcio.`<base_hostname>`
+* https://tsa.`<base_hostname>`
+* https://tuf.`<base_hostname>`
 
-Utilize the following steps to sign a container that has been published to an OCI registry
-
-1. Export the following environment variables substituting `BASE_HOSTNAME`, `KEYCLOAK_URL` and if necessary also `KEYCLOAK_REALM` with the values used as part of the provisioning:
-
-```shell
-export BASE_HOSTNAME="TODO-provide-base-hostname"
-export KEYCLOAK_URL="TODO-your-keycloak-url"
-export KEYCLOAK_REALM=trusted-artifact-signer
-
-export TUF_URL=https://tuf.$BASE_HOSTNAME
-export OIDC_ISSUER_URL=$KEYCLOAK_URL/auth/realms/$KEYCLOAK_REALM
-export COSIGN_FULCIO_URL=https://fulcio.$BASE_HOSTNAME
-export COSIGN_REKOR_URL=https://rekor.$BASE_HOSTNAME
-export COSIGN_MIRROR=$TUF_URL
-export COSIGN_ROOT=$TUF_URL/root.json
-export COSIGN_OIDC_CLIENT_ID=$KEYCLOAK_REALM
-export COSIGN_OIDC_ISSUER=$OIDC_ISSUER_URL
-export COSIGN_CERTIFICATE_OIDC_ISSUER=$OIDC_ISSUER_URL
-export COSIGN_YES="true"
-export SIGSTORE_FULCIO_URL=$COSIGN_FULCIO_URL
-export SIGSTORE_OIDC_ISSUER=$COSIGN_OIDC_ISSUER
-export SIGSTORE_REKOR_URL=$COSIGN_REKOR_URL
-export REKOR_REKOR_SERVER=$COSIGN_REKOR_URL
-```
-
-2. Initialize the TUF roots
-
-```shell
-cosign initialize
-```
-
-Note: If you have used `cosign` previously, you may need to delete the `~/.sigstore` directory
-
-3. Sign the desired container
-
-```shell
-cosign sign -y <image>
-```
-
-Authenticate with the Keycloak instance using the desired credentials.
+## Prerequisites
 
-4. Verify the signed image
+* RHEL x86\_64 9.2 or greater.
+* Command-line access to the Ansible control node with a user that has `sudo` privileges.
+* Installation and configuration of Ansible on a control node to perform the automation.
+* Installation of the Ansible collections on the control node, unless installing from the Ansible Automation Hub.
+  > [!NOTE]
+  To install the required dependencies automatically, you can run the `ansible-galaxy install` command.
```

Comment:
This should be `ansible-galaxy install redhat.artifact_signer` (the name `redhat.artifact_signer` could still change, but it's very likely to be this)

Reply:
Good to know, thank you!